### ISSUE TITLE
Support for managing KubeVirt and connecting to VMs?

### ISSUE THREAD

**Issue by victorhooi** [lensapp/lens#3493]
**What would you like to be added**:

Support for managing KubeVirt VMs, and connecting via serial/VNC to them

**Why is this needed**:

There doesn't appear to be any good GUI-based options to connect to and manage KubeVirt VMs:

https://kubevirt.io/2019/KubeVirt_UI_options.html

(Outside of the OpenShift specific ones).

It would be really awesome if Lens could support this!

**Environment you are Lens application on:**

- Kubernetes distribution: Any
- Desktop OS: Any

**Comment by Mihai-CMM** [comment]
Yep will like this feature also. Is it on your roadmap? I am evaluating Lens at the moment and even though this is not deal breaker it will be a heavy feature for us since kubevirt is planed to be heavily used

**Comment by X-Siro** [comment]
Yeah, it might be killer feature
